content/docs/command-reference/exp/run.md

```diff
@@ -109,18 +110,21 @@ execution. `dvc exp show` will mark queued experiments with an asterisk `*`.
 > Note that queuing an experiment that uses checkpoints implies `--reset`,
 > unless a `--rev` is provided (refer to the previous section).
 
-Use `dvc exp run --run-all` to process the queue. Adding `-j` (`--jobs`),
-experiment queues can be run in parallel for better performance. This creates a
-temporary workspace copy for each subprocess (in `.dvc/tmp/exps`). See also the
-`--temp` option. Queued (and temporary) experiment runs do not get applied to
-the workspace like regular ones.
+Use `dvc exp run --run-all` to process the queue. This is done outside your
+<abbr>workspace</abbr> (in a temporary dir in `.dvc/tmp/exps`), so you can
+continue working on the project meanwhile (e.g. on another terminal). You can
+run a single experiment outside the workspace with `dvc exp run --temp`.
+
+Adding `-j` (`--jobs`), experiment queues can be run in parallel for better
+performance (creates a tmp dir for each subprocess).
```

Comment:
```suggestion
performance (creates a tmp dir for each experiment).
```

Reply:
> creates a tmp dir for each experiment

This is always true, even without `--jobs`. Changed to "jobs" instead (c0cbf1b5).